tools/text_extractor/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Add your dependencies here
 
 # Required for all unstract tools
-unstract-sdk~=0.35.0
+unstract-sdk~=0.37.0
```

Comment:
Here also, are we missed to update `0.36.0` ?

Reply:
@chandrasekharan-zipstack any idea why?

Reply:
Changes made in 0.36.0 is not relavant to text-extractor. That's why it is not updated.